diff --git a/lib/python/Components/Renderer/DMCHDMaxTemp.py b/lib/python/Components/Renderer/DMCHDMaxTemp.py
index 4bfd5168e..ea0e01e5b 100644
--- a/lib/python/Components/Renderer/DMCHDMaxTemp.py
+++ b/lib/python/Components/Renderer/DMCHDMaxTemp.py
@@ -53,7 +53,7 @@ class DMCHDMaxTemp(Renderer, VariableText):
 							maxtemp = tt
 				except:
 					pass
-				self.text = str(maxtemp) + "°C"
+				self.text = str(maxtemp)
 			else:
 				loada = 0
 				try:
diff --git a/lib/python/Components/Renderer/MaxTempNobile.py b/lib/python/Components/Renderer/MaxTempNobile.py
index c60327a97..361f7ac9a 100644
--- a/lib/python/Components/Renderer/MaxTempNobile.py
+++ b/lib/python/Components/Renderer/MaxTempNobile.py
@@ -53,7 +53,7 @@ class MaxTempNobile(Renderer, VariableText):
 							maxtemp = tt
 				except:
 					pass
-				self.text = str(maxtemp) + "°C"
+				self.text = str(maxtemp)
 			else:
 				loada = 0
 				try:
diff --git a/lib/python/Components/Renderer/valioOledInfo.py b/lib/python/Components/Renderer/valioOledInfo.py
index 2a079b7ab..051555611 100644
--- a/lib/python/Components/Renderer/valioOledInfo.py
+++ b/lib/python/Components/Renderer/valioOledInfo.py
@@ -53,7 +53,7 @@ class valioOledInfo(Renderer, VariableText):
 							maxtemp = tt
 				except:
 					pass
-				self.unten = str(maxtemp) + " °C"
+				self.unten = str(maxtemp)
 			elif self.Zaehler == 21:
 				self.oben = "loads:"
 				loada = 0
diff --git a/lib/python/Components/Renderer/valioPosition.py b/lib/python/Components/Renderer/valioPosition.py
index b298c505d..4ff56261b 100644
--- a/lib/python/Components/Renderer/valioPosition.py
+++ b/lib/python/Components/Renderer/valioPosition.py
@@ -28,9 +28,9 @@ class valioPosition(Renderer, VariableText):
 					if (frontendData.get("tuner_type") == "DVB-S"):
 						orbital_pos = int(frontendData["orbital_position"])
 						if orbital_pos > 1800:
-							orb_pos = str((float(3600 - orbital_pos)) / 10.0) + "°W"
+							orb_pos = str((float(3600 - orbital_pos)) / 10.0)
 						elif orbital_pos > 0:
-							orb_pos = str((float(orbital_pos)) / 10.0) + "°E"
+							orb_pos = str((float(orbital_pos)) / 10.0)
 					elif (frontendData.get("tuner_type") == "DVB-T"):
 						orb_pos = "DVB-T"
 					elif (frontendData.get("tuner_type") == "DVB-C"):
diff --git a/lib/python/Components/Renderer/valioSystem.py b/lib/python/Components/Renderer/valioSystem.py
index 663e84af4..0b95afcc8 100644
--- a/lib/python/Components/Renderer/valioSystem.py
+++ b/lib/python/Components/Renderer/valioSystem.py
@@ -53,7 +53,7 @@ class valioSystem(Renderer, VariableText):
 			except:
 				pass
 			if self.ZeigeTemp:
-				self.text = "cpu " + loada + "\ntmp " + str(maxtemp) + "°C\nfan " + str(int(fan / 2))
+				self.text = "cpu " + loada + "\ntmp " + str(maxtemp) + "\nfan " + str(int(fan / 2))
 			else:
 				self.text = "cpu\n" + loada
 
diff --git a/lib/service/Makefile.inc b/lib/service/Makefile.inc
index 520b5edc6..ccc7a454b 100644
--- a/lib/service/Makefile.inc
+++ b/lib/service/Makefile.inc
@@ -44,9 +44,9 @@ serviceinclude_HEADERS += \
 	service/servicexine.h
 endif
 
-if HAVE_LIBDDVD
-service_libenigma_service_a_SOURCES += \
-	service/servicedvd.cpp
-serviceinclude_HEADERS += \
-	service/servicedvd.h
-endif
+#if HAVE_LIBDDVD
+#ervice_libenigma_service_a_SOURCES += \
+#	service/servicedvd.cpp
+#erviceinclude_HEADERS += \
+#	service/servicedvd.h
+#ndif
diff --git a/lib/service/servicedvd.cpp b/lib/service/servicedvd.cpp
index 92565cf70..895399cf0 100644
--- a/lib/service/servicedvd.cpp
+++ b/lib/service/servicedvd.cpp
@@ -562,7 +562,7 @@ RESULT eServiceDVD::setSlowMotion(int ratio)
 	// pass ratio as repeat factor.
 	// ratio=2 means 1/2 speed
 	// ratio=3 means 1/3 speed
-	ddvd_send_key(m_ddvdconfig, ratio < 0 ? DDVD_KEY_SLOWBWD : DDVD_KEY_SLOWFWD);
+	//ddvd_send_key(m_ddvdconfig, ratio < 0 ? DDVD_KEY_SLOWBWD : DDVD_KEY_SLOWFWD);
 	ddvd_send_key(m_ddvdconfig, ratio);
 	return 0;
 }
@@ -570,7 +570,7 @@ RESULT eServiceDVD::setSlowMotion(int ratio)
 RESULT eServiceDVD::setFastForward(int trick)
 {
 	eDebug("[eServiceDVD] setTrickmode: %d", trick);
-	ddvd_send_key(m_ddvdconfig, trick < 0 ? DDVD_KEY_FASTBWD : DDVD_KEY_FASTFWD);
+	//ddvd_send_key(m_ddvdconfig, trick < 0 ? DDVD_KEY_FASTBWD : DDVD_KEY_FASTFWD);
 	ddvd_send_key(m_ddvdconfig, trick);
 	return 0;
 }
diff --git a/po/Makefile.am b/po/Makefile.am
index 0093e6782..e48977b2f 100644
--- a/po/Makefile.am
+++ b/po/Makefile.am
@@ -9,6 +9,7 @@ EXTRA_DIST = $(LANGPO)
 # the TRANSLATORS: allows putting translation comments before the to-be-translated line.
 enigma2-py.pot: $(top_srcdir)/*.py $(top_srcdir)/lib/python/*/*.py $(top_srcdir)/lib/python/*/*/*.py $(top_srcdir)/lib/python/Plugins/*/*/*.py
 	$(XGETTEXT) --no-wrap -L Python --from-code=UTF-8 -kpgettext:1c,2 --add-comments="TRANSLATORS:" -d @PACKAGE_NAME@ -s -o $@ $^
+	$(SED) --regexp-extended --in-place $@ --expression="s/(#:\s).*\/(.*\/plugin\.py|[^.*\/plugin\.py].*)(:\d*)/\1\2\3/"
 
 enigma2-xml.pot: $(srcdir)/xml2po.py $(top_srcdir)/data/*.xml $(top_srcdir)/lib/python/Plugins/SystemPlugins/*/*.xml
 	$(PYTHON) $^ > $@
